timescaledb/page-index.js

```diff
@@ -19,6 +19,10 @@ module.exports = [
       {
         title: "Contribute to TimescaleDB",
         href: "contribute-to-timescaledb"
+      },
+      {
+        title: "Promscale",
+        href: "promscale"
```

Comment:
My opinion is that we create this as a top-level volume like TimescaleDB, API, etc. in this branch and work with the Observatility team to move over docs that they feel are relevant. The "tutorials" for Promscale were always intended/meant to be moved here too, so doing that would give us a few pages/sections to start and then a few from GitHub repo.

With a week or so of time, we should be able to get a bunch of them moved and then we can just merge and create the new section as a "release" of sorts.

But, putting it here just isn't the right place and we'd prefer (I think) Google et. al. to index/search all of this "correctly" from the start, right?

Reply:
OK, no worries. I'll update 👍

Reply:
OK, created a new top level section. If someone could check my YAML, that would be great (and there might be some other frontend steps?). I'm not sure I agree with breaking up the tutorials at this stage, though.